python/tvm/relay/op/_tensor.py

```diff
@@ -87,6 +87,10 @@
 register_broadcast_schedule("fast_exp")
 register_broadcast_schedule("fast_tanh")
 register_broadcast_schedule("fast_erf")
+# a fake on_device schedule.
+# this will not be used in actual computation
+# as on_device will be removed during DeviceAnnotation pass
+register_injective_schedule("on_device")
```

Comment:
@zhanghaohit what happens if we don't register this annotation op schedule?

Reply:
It will raise 
`
AssertionError: on_device doesn't have FTVMStrategy registered
`
during build_module.cc::Optimize before we do the `RunDeviceAnnotationPass`.